f2/src/components/warning-banner/index.js

```diff
@@ -10,7 +11,7 @@ export const WarningBanner = props => {
   return (
     <Alert p={4} status={status} bg={`${bg}.400`} fontFamily="body">
       <AlertIcon mt={0} color="black" />
-      <AlertDescription color="black">{message}</AlertDescription>
+      <AlertDescription color="black"><Trans id="banner.warning" /></AlertDescription>
```

Comment:
Replace the Trans component with props.children. App.js already contains The exact same Trans tag as a child of the warning banner component!

then 🚢

Reply:
*also remove the import to prevent lint error

Reply:
Done - thank you for this 👍